src/jiti.ts

```diff
@@ -49,6 +50,7 @@ const defaults: JITIOptions = {
   alias: _EnvAlias,
   nativeModules: _EnvNative || [],
   transformModules: _EnvTransform || [],
+  experimentalBun: _ExpBun === undefined ? !!process.versions.bun : !!_ExpBun,
```

Comment:
What do you think about moving Bun detection into https://github.com/unjs/std-env?